includes/Data/Flows.php

```diff
@@ -13,6 +13,8 @@ final class Flows {
 	 * @var array
 	 */
 	protected static $data = array(
+		'version'              => '0.1',
```

Comment:
```suggestion
		'version'              => '1.0.0',
```
Let's start following semantic versioning for this as well?